CHANGELOG.md

```diff
@@ -6,6 +6,9 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+### Added
+- Added a RedHat-certified version of the image [cyberark/secrets-provider-for-k8s](https://github.com/cyberark/secrets-provider-for-k8s/pull/93)
```

Comment:
the word `Added` is redundant

Reply:
also you are missing the issue number in the link. It should be `cyberark/secrets-provider-for-k8s#93`